tests/unit/test_platform.cpp

```diff
@@ -594,4 +594,20 @@ OT_TOOL_WEAK void otPlatDsoDisconnect(otPlatDsoConnection *aConnection, otPlatDs
 
 #endif // #if OPENTHREAD_CONFIG_DNS_DSO_ENABLE
 
+OT_TOOL_WEAK int SEGGER_RTT_SetFlagsUpBuffer(unsigned BufferIndex, unsigned Flags)
```

Comment:
Why do we need these in `test_platform`?

Reply:
The power calibration is located in the platform/utils. When building the unit test of the power calibration, it will build the `logging_rtt.c` at the same time. These functions are defined here to avoid building failure.

I have moved the power calibration implementation to core/utils, we do not need these functions any more.

Reply:
The power calibration is located in the platform/utils. When building the unit test of the power calibration, it will build the `logging_rtt.c` at the same time. These functions are defined here to avoid building failure.

I have moved the power calibration implementation to core/utils, we do not need these functions anymore.

Reply:
The power calibration is located in the platform/utils. When building the unit test of the power calibration, it will build the `logging_rtt.c` at the same time. These functions are defined here to avoid building failure.

I have moved the power calibration implementation to core/utils, we do not need these functions anymore.